### DISCUSSION TITLE
Have EVERY email to one person in ONE conversation?

### DISCUSSION THREAD

**Discussion by Johneltonsbrain** [protz/thunderbird-conversations#1650] [Q&A]
I think once the other person doesnt reply to my Email but rather sends an independent one, it starts a new conversation.

I have looked in the settings but cant find an option.

<3

**Comment by Standard8** [comment]
If the other person starts a new message thread there isn't much Thunderbird can do about it - the references between messages are lost.

In any case, Thunderbird itself is handling the conversations mapping, rather than Conversations. So this would really be a Thunderbird issue to resolve.
